gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -828,7 +840,7 @@ def IdentifyUseSpell ():
 		ItemInfoWindow.Unload ()
 	GemRB.ChangeItemFlag (pc, slot, IE_INV_ITEM_IDENTIFIED, OP_OR)
 	GemRB.PlaySound(DEF_IDENTIFY)
-	OpenItemInfoWindow(slot)
+	OpenItemInfoWindow(slot=slot)
```

Comment:
Codacy found an issue: [No value for argument 'btn' in function call](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)